src/js/correlation/models/Correlation.ts

```diff
@@ -32,10 +32,10 @@ export class Correlation {
         if (data instanceof zng.Primitive) {
           return data.toString()
         } else {
-          return data
-            .getValue()
-            .map((v) => v.toString())
-            .join(" ")
+          const value = data.getValue()
+          if (!value) return null
+          if (isString(value)) return value
```

Comment:
this wasn't actually the main issue, but it's safer code now so I'm including it